tests/update.rs

```diff
@@ -94,7 +94,7 @@ fn api_insert() {
                 let key = format!("key {}", n);
                 let value = format!("value {}", n);
 
-                db.try_insert(key.as_bytes(), value.as_bytes())
+                db.try_insert(key.as_bytes().to_vec(), value.as_bytes().to_vec())
```

Comment:
This becomes a bit ugly for normal user use...   Will Into<[u8]> improve user ergonomics?